great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -277,67 +252,54 @@ def _build_parameters(
             variables=variables,
             parameters=parameters,
         )
-        metric_values: MetricComputationValues = metric_computation_result.metric_values
+        metric_values: np.ndarray = metric_computation_result.metric_values
         details: MetricComputationDetails = metric_computation_result.details
 
-        truncate_values: Dict[str, Number] = self._get_truncate_values_using_heuristics(
-            metric_values=metric_values,
+        # Obtain sampling_method directive from "rule state" (i.e., variables and parameters); from instance variable otherwise.
+        sampling_method: str = get_parameter_value_and_validate_return_type(
             domain=domain,
+            parameter_reference=self._sampling_method,
+            expected_return_type=str,
             variables=variables,
             parameters=parameters,
         )
-        lower_bound: Optional[float] = truncate_values.get("lower_bound")
-        upper_bound: Optional[float] = truncate_values.get("upper_bound")
+        if not (
+            sampling_method
+            in NumericMetricRangeMultiBatchParameterBuilder.RECOGNIZED_SAMPLING_METHOD_NAMES
+        ):
```

Comment:
Nitpick - Can we use `if sampling_method not in`? I think that's more Pythonic

Reply:
@cdkini Fixed (in next commit).